dev/suites/xtrigger/kafka/consumer/suite.rc

```diff
@@ -0,0 +1,52 @@
+#!Jinja2
+
+#------------------------------------------------------------------------------
+# A suite that uses the cylc_kafka_consumer module as an external trigger
+# function, to trigger tasks off of Kafka messages. Jinja2 inputs are defined
+# in rose-suite.conf.
```

Comment:
This suite is causing `./tests/validate/04-builtin-suites.t` to fail if `kafka` is not installed.

Reply:
Sorry, I meant to move this to another location. It should probably be an example of how to trigger off of Kafka messages, rather than an officially supported interface.